src/Uno.UI.RemoteControl/HotReload/ClientHotReloadProcessor.MetadataUpdate.cs

```diff
@@ -206,31 +201,58 @@ private static async void ReloadWithUpdatedTypes(Type[] updatedTypes)
 				// This is called before the visual tree is updated
 				_ = handlerActions?.Do(h => h.Value.BeforeVisualTreeUpdate(updatedTypes)).ToArray();
 
-				// Iterate through the visual tree and either invole ElementUpdate, 
-				// or replace the element with a new one
-				foreach (
-					var (element, elementHandler, elementMappedType) in
-					EnumerateHotReloadInstances(
+				var capturedStates = new Dictionary<string, IDictionary<string, object>>();
```

Comment:
The value does not need to be an interface in this case.

Reply:
Updated